pkgs/applications/editors/emacs/macport-25.1.nix

```diff
@@ -48,7 +48,7 @@ stdenv.mkDerivation rec {
     "--enable-mac-app=$$out/Applications"
   ];
 
-  CFLAGS = "-O3 -DMAC_OS_X_VERSION_MAX_ALLOWED=1090";
+  CFLAGS = "-O3 -DMAC_OS_X_VERSION_MAX_ALLOWED=1090 -DMAC_OS_X_VERSION_MIN_REQUIRED=1090";
```

Comment:
Well we can't have MAC_OS_X_VERSION_MAX_ALLOWED without MAC_OS_X_VERSION_MIN_REQUIRED because max allowed can't be less than min required. Why MAC_OS_X_VERSION_MAX_ALLOWED is necessary is unclear but it seems to fail without it.